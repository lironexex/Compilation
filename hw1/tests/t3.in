// unprintable char:
var num = 1.0e+6 + 
