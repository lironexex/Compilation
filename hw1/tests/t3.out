1 COMMENT 1
2 VAR var
2 ID num
2 ASSIGN =
2 DEC_REAL 1.0e+6
2 BINOP +
Error 
